.github/workflows/misc/set_vs_static_runtime_libraries.py

```diff
@@ -0,0 +1,32 @@
+from xml.etree import ElementTree
+import sys
+
+ElementTree.register_namespace("", "http://schemas.microsoft.com/developer/msbuild/2003")
+
+
+def adapt(in_path, out_path):
+    tree = ElementTree.parse(in_path)
+
+    root = tree.getroot()
+    for item_definition_group in root.findall('{http://schemas.microsoft.com/developer/msbuild/2003}ItemDefinitionGroup'):
+        condition = item_definition_group.attrib["Condition"]
+        is_release = condition.lower().find("release") > -1
+        is_debug = condition.lower().find("debug") > -1
+        assert is_release ^ is_debug, condition
+
+        cl_args = item_definition_group.findall('{http://schemas.microsoft.com/developer/msbuild/2003}ClCompile')
+        assert len(cl_args) == 1
+        cl_args = cl_args[0]
+
+        runtime_library = ElementTree.Element("RuntimeLibrary")
+        runtime_library.text = "MultiThreaded"
+        if is_debug:
+            runtime_library.text += "Debug"
+        cl_args.append(runtime_library)
+
+    tree.write(out_path)
+
+
+if __name__ == "__main__":
+    assert len(sys.argv) == 3
```

Comment:
You could use `_, in_path, out_path = sys.argv` instead and remove the assertion.